client/index.jsx

```diff
@@ -19,17 +22,31 @@ class App extends React.Component {
     $.get({
       url: `/description?prod_id=${prod}`,
       success: (res) => {
-        console.log(res)
         this.setState({
           data:res
         })
       },
       dataType: 'JSON'
     })
   }
-
+//conditional rendering here is a work around for an asycn bug that was crashing the app.  The first render had nothing in state so was passing undefined.  Now code waits for get request res before rendering components
   render() {
-    return <div>Hello React,Webpack 4 & Babel 7!</div>
+    if (this.state.data.length < 1) {
+      return <p>loading</p>
+    }
+    if (this.state.page === 1) {
+      return (
+        <div>
+          <ShipPay payment={this.state.data[1]}/>
+        </div>
+      )
+    }
+    return (
+      <div>
+        <Description prodInfo={this.state.data[0]}/>
+      </div>
+
```

Comment:
No functional difference, but could make for more condensed code to put your this.state.page conditional inside the container <div> to switch between your ShipPay and Description components instead of needing two completely separate return statements.